cirq-google/cirq_google/json_resolver_cache.py

```diff
@@ -77,4 +77,5 @@ def _old_xmon(*args, **kwargs):
         # pylint: enable=line-too-long
         'cirq.google.EngineResult': cirq_google.EngineResult,
         'cirq.google.GridDevice': cirq_google.GridDevice,
+        'cirq.google.GoogleCZTargetGateset': cirq_google.GoogleCZTargetGateset,
```

Comment:
Why is the key `cirq.google.GoogleCZTargetGateset` instead of `cirq_google.GoogleCZTargetGateset`?

Reply:
@mpharrigan do you happen to have the context?

Reply:
I started adding namespaces to my json cirq_types before the great `cirq.google -> cirq_google` migration. It's not worth it to change the old ones. I continued adding them with the dot for consistency.

Reply:
Okay, let's leave it like this for consistency. Thanks for the context.